### ISSUE TITLE
fix[dev]: ✨ adjust sd-textarea disabled text color

### ISSUE THREAD

**Issue by paulovareiro29** [solid-design-system/solid#1661]
## Description
The disabled text color on the sd-textarea, in the Figma docs is currently `neutral-500` while in the implementation it is `neutral-700`

Ticket created from #1642

### Suggested Solution
Disabled text color is adjusted to `neutral-500` on implementation

### Technical Information
Any text content on the sd-textarea should be `neutral-500`

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by coraliefeil** [comment]
@paulovareiro29 I think this is not a feat issue but a bug issue which can be moved to ready column.

**Comment by paulovareiro29** [comment]
Aligned with @MartaPintoTeixeira and it was correct. No need to update colors. Placeholder color should remain neutral-700 even when disabled.
